sdk/formrecognizer/ai-form-recognizer/review/ai-form-recognizer.api.md

```diff
@@ -254,6 +266,13 @@ export interface FormRecognizerOperationOptions extends OperationOptions {
 // @public
 export type FormRecognizerRequestBody = Blob | ArrayBuffer | ArrayBufferView | NodeJS.ReadableStream;
 
+// @public
+export interface FormSelectionMark extends FormElementCommon {
+    confidence?: number;
```

Comment:
same here, in which cases it is nullable?

Reply:
I thought we decided to give it 1, if it's missing.